desktop/core/src/desktop/lib/botserver/views_tests.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import unittest
+import sys
+
+from nose.tools import assert_equal, assert_true
+from django.test import TestCase
+from desktop.lib.botserver.views import *
+
+if sys.version_info[0] > 2:
+  from unittest.mock import patch
+else:
+  from mock import patch
+
+LOG = logging.getLogger(__name__)
+
+class TestBotServer(unittest.TestCase):
+  def test_get_bot_id(self):
```

Comment:
Currently, the test is failing.
Need to know where am I going wrong in the mocking of the Client or something else @romainr ?

<img width="1381" alt="Screenshot 2021-02-15 at 9 37 33 PM" src="https://user-images.githubusercontent.com/42064744/107969639-073c2a00-6fd6-11eb-958e-9cc84944e22c.png">